src/mote/main.py

```diff
@@ -37,7 +42,12 @@
 from mote.modules.late import fetch_recent_meetings
 
 main = Flask(__name__)
+socketio = SocketIO(main)
 recognition_pattern = "(.*)[\-\.]([0-9]{4}-[0-9]{2}-[0-9]{2})-([0-9]{2}\.[0-9]{2})"
+thread = None
+thread_lock = Lock()
+client_count = 0
+exit = Event()
```

Comment:
`exit` is already used internally by Python, choose another name, like `exit_thread`.